validator/bin/3d-tiles-validator.js

```diff
@@ -1,2 +1,15 @@
 #!/usr/bin/env node
 'use strict';
+
+var argv = require('yargs').argv;
+var fsExtra = require('fs-extra');
+var path = require('path');
+var Promise = require('bluebird');
+
+var readTileset = require('../lib/readTileset');
+var validateTileset = require('../lib/validateTileset');
+
+var filePath = argv._[0];
+var jsonTileset = readTileset(filePath);
+var response = validateTileset(jsonTileset);
+console.log(response);
```

Comment:
`readTileset` and `validateTileset` are both promises, they don't return immediately.

Instead of just printing the response, print `response.message`.

Also add newline.